driver/src/models.rs

```diff
@@ -107,7 +107,7 @@ fn merkleize(leafs: Vec<Vec<u8>>) -> H256 {
     merkleize(next_layer)
 }
 
-#[derive(Clone, Deserialize, Eq, Ord, PartialEq, PartialOrd)]
+#[derive(Clone, Deserialize, Eq, Ord, PartialEq, PartialOrd, Debug)]
```

Comment:
Why is this needed? We should probably prefer to implement Display properly ourselves.

Reply:
This was only temporary while I "debug" my code.